test/size.txt

```diff
@@ -25,37 +25,37 @@
 167.24 kB   |    10.67 kB   |     4.68 kB   |   v0/amp-facebook-0.1.js
  43.47 kB   |     3.11 kB   |     1.38 kB   |   v0/amp-fit-text-0.1.js
 134.42 kB   |     4.05 kB   |     1.82 kB   |   v0/amp-font-0.1.js
-207.58 kB   |    14.36 kB   |     5.39 kB   |   v0/amp-form-0.1.js
+207.57 kB   |    14.36 kB   |     5.39 kB   |   v0/amp-form-0.1.js
 155.68 kB   |     3.49 kB   |     1.65 kB   |   v0/amp-fresh-0.1.js
  53.53 kB   |     4.22 kB   |     1.81 kB   |   v0/amp-fx-flying-carpet-0.1.js
  34.36 kB   |     1.55 kB   |       790 B   |   v0/amp-gfycat-0.1.js
  63.29 kB   |     3.78 kB   |     1.73 kB   |   v0/amp-google-vrview-image-0.1.js
 189.03 kB   |    10.81 kB   |     4.48 kB   |   v0/amp-iframe-0.1.js
-266.21 kB   |    27.59 kB   |     9.02 kB   |   v0/amp-image-lightbox-0.1.js
+266.42 kB   |    27.63 kB   |     9.02 kB   |   v0/amp-image-lightbox-0.1.js
  60.57 kB   |     2.75 kB   |     1.29 kB   |   v0/amp-instagram-0.1.js
-121.87 kB   |      5.4 kB   |     2.52 kB   |   v0/amp-install-serviceworker-0.1.js
+121.87 kB   |      5.4 kB   |     2.51 kB   |   v0/amp-install-serviceworker-0.1.js
  41.57 kB   |     2.58 kB   |     1.21 kB   |   v0/amp-jwplayer-0.1.js
  78.22 kB   |     3.67 kB   |     1.62 kB   |   v0/amp-kaltura-player-0.1.js
 173.39 kB   |     8.21 kB   |     3.13 kB   |   v0/amp-lightbox-0.1.js
 172.59 kB   |    12.51 kB   |     4.22 kB   |   v0/amp-lightbox-viewer-0.1.js
-126.88 kB   |     2.96 kB   |      1.4 kB   |   v0/amp-list-0.1.js
-184.37 kB   |     9.21 kB   |     3.59 kB   |   v0/amp-live-list-0.1.js
+126.93 kB   |     3.31 kB   |     1.56 kB   |   v0/amp-list-0.1.js
```

Comment:
Are these increased because size.txt hasn't been updated before or because of this change?